### ISSUE TITLE
Implement share bar

### ISSUE THREAD

**Issue by entrity** [entrity/tipsy-web-public#19]
Right now all the buttons are just links to '#'

**Comment by sirtastic** [comment]
@entrity I don't think this should be in initial launch unless you think it would be an easy/quick implementation.

Visually this will be much like Yummly.com execution:

Favorites directory:
![image](https://cloud.githubusercontent.com/assets/3707868/8896937/d85851b2-33bf-11e5-914c-5473c56fbf53.png)

Individual list:
![image](https://cloud.githubusercontent.com/assets/3707868/8896941/ede1eae8-33bf-11e5-998d-c79ceaa0db27.png)

**Comment by sirtastic** [comment]
Need to assign this to you since it involves pulling content form page into the header. 

We need to apply open graph markup to the drink pages.  This needs to be in the header for the drink pages and needs to dynamically pull content from the individual drink titles/descriptions/etc. 

I've included sample content below. This all needs to be replaced with the pulled content.

```
<meta property="og:url"                      content="http://tipsyology.com/drinks/1926" />
<meta property="og:site_name"         content="Tipsyology - Discover, create and share all things tipsy!">
<meta property="og:type"                   content="recipe" />
<meta property="og:title"                    content="Vodkatini cocktail" />
<meta property="og:description"        content="How to make a Vodkatini" />
<meta property="og:image"                content="http://tipsyology.com/images/drinks/vodktatini.jpeg" />
```

**og:url** we will want the URL to be something like **http://tipsyology.com/cocktails/vodkatini**. not /drinks/### for SEO reasons. I'll make a separate ticket for this.
**og:site_name** This stays the same, don't modify.
**og:type**  does not need to change, it should stay as recipe.
**og:title**   This should be drink name + cocktail
**og:description**   This should pull the description if one is present. Otherwise the fallback should be "How to make a Vodkatini cocktail.
**og:image**  This should link to image if one is present. Otherwise we should have a fallback default.  Use this image as the fallback:

![facebook-profile](https://cloud.githubusercontent.com/assets/3707868/8897369/10e8f1d6-33c9-11e5-9eb3-4a59e56b2d04.png)

Once the open graph is in place I'll implement the links for the share buttons.

When this is complete assign to me. I'll implement printing stylesheet/button.

Last, email button. This will pop a modal that will email a link + input fields to emails that can be separated by commas. I will build this modal. You'll need to plug it in.

**Comment by entrity** [comment]
@sirtastic,

1. Done. Assigning to you.
2. You don't need me to plug in the email link if you don't mind using a `mailto` url. (Saves a chunk of programming time.) Use the form on this page: http://www.cha4mot.com/t_mailto.html. All of the fields are optional, of course, with the exception of the 'to' field. Click 'Create URL', then copy the `mailto` url out of the field below.

*Handled by commit 97234464ec4141bc6e666a1ec54f89b974904341*

**Comment by sirtastic** [comment]
Can't test this since facebook can't crawl my localhost to pull the data.

Turns out twitter has their own open graph markup. When you have time add the following:

```
<!-- for Twitter -->          
<meta name="twitter:card" content="summary" />
<meta name="twitter:title" content="" />
<meta name="twitter:description" content="" />
<meta name="twitter:image" content="" />
```

**Comment by entrity** [comment]
commit fda129f04a6168954df06ae80cd312873871aba5
Author: Markham Anderson <quinesquines@gmail.com>
Date:   Sun Aug 2 10:33:25 2015 -0700

    Add twitter meta tags. Resolves #19 (github)

**Comment by entrity** [comment]
@sirtastic : back to you.
